source/gfoidl.Base64/Avx2Helper.cs

```diff
@@ -0,0 +1,83 @@
+﻿#define AVX_PERMUTE
+//-----------------------------------------------------------------------------
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+
+namespace gfoidl.Base64
+{
+    internal static class Avx2Helper
+    {
+        // https://github.com/dotnet/coreclr/issues/21105
+        private static readonly bool s_macHack = RuntimeInformation.IsOSPlatform(OSPlatform.OSX);
+        //---------------------------------------------------------------------
+        static Avx2Helper()
+        { }
+        //---------------------------------------------------------------------
+        public static int DummyToInitType(int a, int b) => a + b;
```

Comment:
This hack will be undone once the `PlatformNotSupportedException` is elided for MacOS.